boards/README.md

```diff
@@ -4,36 +4,79 @@ Platforms Supported by Tock
 The `/boards` directory contains the physical hardware platforms
 that Tock supports.
 
+Tock divides boards into three approximate 'tiers' of support.
+These tiers are newly defined and are a bit informal as a result,
+but the approximate definitions:
+
+ - **Tier 1:** The most feature-complete and thoroughly tested boards.
+               Loosely, these are boards used most regularly by core
+               team members or other highly engaged contributors.
+ - **Tier 2:** Platforms seeing reasonably regular use. These generally
+               have broader, but still incomplete, peripheral support.
+               They may also be 'relatives' of Tier 1 boards (e.g. a
+               less-used varient in the nrf52 family) – likely in good
+               shape, but not heavily tested. Some Tier 2 boards may
+               have known issues, which are documented in release notes
+               during release testing.
+ - **Tier 3:** New or highly experimental. These should support the
+               minimum platform requirements laid out in
+               [the Porting documentation](../doc/Porting.md), but
+               make no promises beyond that.
+
+ - **Tier Z:** These are unstable _platforms_, where Tock bringup is
+               happening in parallel with hardware development. These
+               are highly experimental by their nature.
+
+### Tier 1
+
 | Board                                                             | Architecture     | MCU            | Interface  | App deployment              | QEMU Support? |
 |-------------------------------------------------------------------|------------------|----------------|------------|-----------------------------|---------------|
 | [Hail](hail/README.md)                                            | ARM Cortex-M4    | SAM4LC8BA      | Bootloader | tockloader                  | No            |
 | [Imix](imix/README.md)                                            | ARM Cortex-M4    | SAM4LC8CA      | Bootloader | tockloader                  | No            |
-| [Nordic nRF52-DK](nordic/nrf52dk/README.md)                       | ARM Cortex-M4    | nRF52832       | jLink      | tockloader                  | No            |
 | [Nordic nRF52840-DK](nordic/nrf52840dk/README.md)                 | ARM Cortex-M4    | nRF52840       | jLink      | tockloader                  | No            |
 | [Nordic nRF52840-Dongle](nordic/nrf52840_dongle/README.md)        | ARM Cortex-M4    | nRF52840       | jLink      | tockloader                  | No            |
```

Comment:
Who uses the dongle regularly? I recall that we had some "issues" testing these, at least w.r.t. getting a TagConnect cable. I guess 1-2 maintainers have one? Might be considered Tier 2.

Reply:
Agreed, I think part of being tier 1 is that the book is supported.